### PR TITLE
[auto] pytorch-pr-5846

### PR DESCRIPTION
**Pull request by o8ht88z00f**
This is an auto PR managed by onnxbot for tracking

https://github.com/pytorch/pytorch/pull/5846 Softmax symbolic should account for negative dim

### CODE REVIEW

**Comment by soumith** [comment]
@onnxbot retest this please
